src/main/java/com/github/lookout/metrics/agent/generators/JavaVMGenerator.java

```diff
@@ -0,0 +1,76 @@
+package com.github.lookout.metrics.agent.generators;
+
+import com.timgroup.statsd.StatsDClient;
+import com.yammer.metrics.core.VirtualMachineMetrics;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by rkuris on 5/28/15.
+ */
+public class JavaVMGenerator implements MetricGenerator {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CassandraJMXGenerator.class);
+
+    protected final VirtualMachineMetrics vm = VirtualMachineMetrics.getInstance();
+    private final HashMap<String, Integer> previous_run_times = new HashMap<String, Integer>();
+    private final HashMap<String, Integer> previous_run_counts = new HashMap<String, Integer>();
+
+    private int bytesToMB(double bytes) {
+        return (int)(bytes/(1024*1024));
+    }
+    private int doubleToPct(double pct) {
+        return (int) Math.round(100 * pct);
+    }
+
+    @Override
+    public void generate(StatsDClient client) {
```

Comment:
In order to be consistent with variable names across all generators, can we please use `statsd` instead of `client`. 

Or even better `statsdClient` across all generators. Because when I was reading it for the first time, it took me a minute to figure it out.